### PR TITLE
Swaasa Changes

### PR DESCRIPTION
**Pull request by swetatech24**

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -108,8 +108,8 @@ cron-scheduler-unblock=0 0 0 * * ? *
 
 spring.mail.host=smtp.gmail.com
 spring.mail.port=587
-spring.mail.username=<Enter mail username>
-spring.mail.password=<Enter mail password>
+spring.mail.username=shekhar.shubham014@gmail.com
```
Comment (harimohanr) [inline src/main/resources/application.properties:111]: @swetatech24 You should not put username and password in properties file. Replace this with the place holder

File: src/main/resources/application.properties [hunk 2]
```diff
@@ -20,7 +20,7 @@ spring.http.multipart.max-file-size=1000MB
 
 ##sms details for CHO
 CHOSmsTemplate= CHO UPTSU SMS
-sendSMSUrl= <Enter SMS URL here>
+sendSMSUrl= http://10.208.122.38:8080/commonapi-v1.0/sms/sendSMS
```
Comment (harimohanr) [inline src/main/resources/application.properties:23]: @swetatech24 You should not put the actual URLs in properties file. Replace this with the placeholder. We had discussed this multiple times.

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -160,20 +160,20 @@ logging.logback.rollingpolicy.max-file-size=20KB
 logging.logback.rollingpolicy.max-history=1
 
 ### SMS Gateway URL details
-sms-username=<Enter SMS username>
-sms-password=<Enter SMS password>
-send-message-url=<Enter SMS sending URL here>
+sms-username=amrit
```
Comment (harimohanr) [inline src/main/resources/application.properties:163]: @swetatech24 Replace it with the placeholder. Do not put credentials in properties file.

File: src/main/resources/application.properties [hunk 4]
```diff
@@ -160,20 +160,20 @@ logging.logback.rollingpolicy.max-file-size=20KB
...
+sms-password=Amrit@321
```
Comment (harimohanr) [inline src/main/resources/application.properties:164]: @swetatech24 Replace it with the placeholder. Do not put credentials in properties file.

File: src/main/resources/application.properties [hunk 5]
```diff
@@ -160,20 +160,20 @@ logging.logback.rollingpolicy.max-file-size=20KB
...
 
 ### Email Gateway URL details
-email-gateway-url=<Enter email gateway URL here>
-send-email-url=<Enter email sending URL here>
-email-username=<Enter AMRIT email username>
-email-password=<Enter AMRIT email password>
+email-gateway-url=https://api.unicel.in
+send-email-url=SendSMS/sendmsg.php?
+email-username=AMRIT
+email-password=Psmri@12
```
Comment (harimohanr) [inline src/main/resources/application.properties:176]: @swetatech24 Replace it with the placeholder. Do not put credentials in properties file.

File: src/main/resources/application.properties [hunk 6]
```diff
@@ -188,10 +188,12 @@ max-pool-size=2
...
-everwellUserName = <Enter everwell username>
-everwellPassword = <Enter everwell password>
-amritUserName = <Enter AMRIT username>
-amritPassword = <Enter AMRIT password>
+everwellUserName = Demo_API_SVA
+
+everwellPassword = Demo_API_SVA
+
+amritUserName = 1097user
+amritPassword = Test@123
```
Comment (harimohanr) [inline src/main/resources/application.properties:196]: @swetatech24 Replace it with the placeholder. Do not put credentials in properties file.

**Review by harimohanr — CHANGES_REQUESTED** [review]
@swetatech24 Look at the review comments and correct it.

File: src/main/resources/application.properties (hunk 3 above)
Comment (swetatech24) [reply src/main/resources/application.properties]: All corrections done

**Review by harimohanr — APPROVED** [review]
@swetatech24 Looks okay to me now

**Pull request merged**